### DISCUSSION TITLE
Trouble Syncing

### DISCUSSION THREAD

**Discussion by RuinKnight** [bf-bonkus/breadiquette#22] [General]
I'm having trouble syncing my  code to @bf-bonkus I've been trying to figure it out and I've been following the instructions but cant figure it out, any suggestions?

**Comment by bf-bonkus** [comment]
uuuh have you regenerated your PAT key?
The instructions are kinda outdated too, are you linking to your repo?

**Comment by bf-bonkus** [comment]
https://github.com/RuinKnight/fbla-game-2024/pull/1 yo merge this

**Comment by RuinKnight** [comment]
Thx,  I will try it as soon as I have access to my laptop.

**Comment by bf-bonkus** [comment]
Do you have access to your laptop yet?

**Comment by RuinKnight** [comment]
sry i cant find my charger andd the CPR isnt working  : (
